src/HotChocolate/Core/src/Types/Types/Attributes/SubscribeAndResolveAttribute.cs

```diff
@@ -34,6 +35,13 @@ public sealed class SubscribeAndResolveAttribute
                         ? typeRef.Type.ElementType
                         : null;
 
+                    // Why does typeRef.Type is ISourceStream not work?
```

Comment:
because it is a instance of system.type and not an instance of ISourceStream. `is` compares the instance of a object to a type

Reply:
So is the `typeof(ISourceStream).IsAssignableFrom(typeRef.Type)` type check below all right? It felt kind of code smelly to me :D